content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -0,0 +1,197 @@
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # the JWT token must have issuer with suffix "@example.com"
+        requestPrincipals: ["*@example.com"]
+    - source:
+        hosts: [".another.org", "*.another.org"]
+        # the JWT token must have issuer with suffix "@another.org"
+        requestPrincipals: ["*@another.org"]
+{{< /text >}}
+
+Alternatively, use the following policy if you want to require JWT validation only to the listed hosts but not for any
+other hosts:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # the JWT token must have issuer with suffix "@example.com"
+        notRequestPrincipals: ["*@example.com"]
+    - source:
+        hosts: [".another.org", "*.another.org"]
+        # the JWT token must have issuer with suffix "@another.org"
+        notRequestPrincipals: ["*@another.org"]
+{{< /text >}}
+
+### Namespace isolation
+
+You want to block all traffic from outside the namespace `foo`, in other words, isolate the namespace `foo` from other
+namespaces. This requires to enable mTLS first.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: foo-isolation
+  namespace: foo
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        namespaces: ["foo"]
+{{< /text >}}
+
+If you want to make sure the policy can not be bypassed by other `ALLOW` policies, you can use the `DENY` policy to
+achieve the result:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ns-isolation
+  namespace: foo
+spec:
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notNamespaces: ["foo"]
+{{< /text >}}
+
+### Namespace isolation with ingress exception
+
+You want to block all traffic from outside the namespace `foo` except the traffic from the ingress gateway.
+This requires to enable mTLS first.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ns-isolation-except-ingress
+  namespace: foo
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        namespaces: ["foo"]
+    - source:
+        principals: ["cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"]
+{{< /text >}}
+
+If you want to make sure the policy can not be bypassed by another `ALLOW` policy, you can use the `DENY` policy to
+achieve the result:
```

Comment:
Same here.

Reply:
not quite sure what does this mean, the DENY policy example is used to explicitly deny if the request is from out of the namespace and also not ingress gateway. You don't need any "allow-nothing" policy for this to work.